Azure/sonic-buildimage
# add INSTALLS, LAZY_INSTALLS, DEPENDS targets to image build

**- What I did**
add INSTALLS, LAZY_INSTALLS, DEPENDS targets to image build

1. _INSTALLS defines the packages to be installed in the image
at the build time.
2. _LAZY_INSTALLS defines the packages to be installed in the image
at the first run time
3. _DEPENDS defines the packages to at the build dependency of
the image

**- How I did it**
refactor slave.mk and corresponding image makefile

**- How to verify it**
1. arista aboot image works
2. add _DEPENDS for arista aboot image and verify debian package installed in the sonic-slave

**- Description for the changelog**
add INSTALLS, LAZY_INSTALLS, DEPENDS targets to image build

**- A picture of a cute animal (not mandatory but encouraged)**
